platform/lib/samples/ExampleFile.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+const FileName = require('./FileName');
+const marked = require('marked');
+const fs = require('fs');
+const path = require('path');
+const PREFIX = /(^|\/)\d\d_/g;
+const SRC_DIR = 'src';
+const GITHUB_PREFIX = 'https://github.com/ampproject/amp-by-example/blob/master/' + SRC_DIR;
```

Comment:
This should be changed or removed.
We should no longer point to the old amp-by-example repo.

Reply:
removed a bunch of unused APIs